app/controllers/api/v1/students_controller.rb

```diff
@@ -1,30 +1,35 @@
-class Api::V1::StudentsController < ApplicationController
+# frozen_string_literal: true
 
-  def index
-    render json: StudentSerializer.new(Student.all)
-  end
+module Api
+  module V1
+    class StudentsController < ApplicationController
```

Comment:
Curious why we used this notation vs the API::V1::StudentsController < ApplicationController

Reply:
That was actually an auto correct from Rubocop